### PR TITLE
usage examples

### PR DESCRIPTION
**Pull request by davidplumb**
For now just a high level run through, will ad a low level one once the dev database has been updated

### CODE REVIEW

File: example_runthroughs.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+## how many people are in fulltime employment in wales
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: how many people are in fulltime employment in wales > How many people are in full-time employment in Wales?

File: example_runthroughs.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+First a new user decides what geography they want to query
+
+```sql
+SELECT * FROM c2011_meta.top_level_geographies
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: missing semicolon here

File: example_runthroughs.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+|combination|
+|-|
+|{AGE,DAYPOP,ECOACT,NSSEC,UNIT}|
+|{AGE,DAYPOP,ECOACT,OGRPMIN,UNIT}|
+|{AGE,DAYPOP,ECOACT,PPHALL,UNIT}|
+|{AGE,DAYPOP,ECOACT,RELIG,UNIT}|
+|{AGE,DAYPOP,ECOACT,TENURE,UNIT,URESPOP}|
+|...
+
+We now have a list of all the possible topic combinations for Wales which we display to the user. The user wants to know how many people are in fulltime employment for the selected gography. So for this instance they select: `economic activity` (ECOACT).
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: fulltime > full-time

File: example_runthroughs.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+|-|-|-|-|
+|QS601SC0010|{AGE:46,ECOACT:570,UNIT:1962}|Age 16 to 74 // Economically active\ Full-time students // Persons|QS601_0_SC_MRG_RCD_AGG|
+|QS601SC0013|{AGE:46,ECOACT:3429,UNIT:1962}|Age 16 to 74 // Persons // Economically inactive\ Student|QS601_0_SC_MRG_RCD_AGG|
+|QS601UK0003|{AGE:46,ECOACT:3823,UNIT:1962}|Age 16 to 74 // Persons // Economically active\ Employee\ Part-time (excluding full-time students)|QS601_0_UK_MRG_RCD_AGG|
+|QS601UK0004|{AGE:46,ECOACT:3795,UNIT:1962}|Age 16 to 74 // Persons // Economically active\ Employee\ Full-time (excluding full-time students)|QS601_0_UK_MRG_RCD_AGG|
+
+
+To get the number of citizens each `name` and `table_name` this corresponds to we can do a query like so:
+```sql
+SELECT geocodeid, QS601SC0010  from QS601_0_SC_MRG_RCD_AGG
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: `SELECT geocodeid, QS601SC0010  from c2011.QS601_0_SC_MRG_RCD_AGG;` (missing db name)

File: example_runthroughs.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+|{AGE,DAYPOP,ECOACT,PPHALL,UNIT}|
+|{AGE,DAYPOP,ECOACT,RELIG,UNIT}|
+|{AGE,DAYPOP,ECOACT,TENURE,UNIT,URESPOP}|
+|...|
+
+Now user selects the `topic` group containing `economic activity` that they desire (we've selected the simplest combination). This means that the results will be grouped by `age` and `economic activity`
+
+`AGE,ECOACT,UNIT`
+
+Finally we now query `variable_combinations` for the relevant `cellnames` and `tablenames` that relate to the data.
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: As there's a stage beyond this would probably remove 'finally'.

File: tables/variables.md [hunk 6]
```diff
@@ -1 +1,71 @@
+# tables/variables
```
Comment (tomgiddingsjisc) [inline tables/variables.md]: Not sure if this file is meant to be here?

Comment (davidplumb) [reply tables/variables.md]: think that ended up in due to not doing a merge, resolved it now

File: example_runthroughs.md (hunk 2 above)
Comment (davidplumb) [reply example_runthroughs.md]: good spot fixed

File: example_runthroughs.md (hunk 3 above)
Comment (davidplumb) [reply example_runthroughs.md]: sure, updated

File: example_runthroughs.md (hunk 4 above)
Comment (davidplumb) [reply example_runthroughs.md]: good spot, fixed

File: example_runthroughs.md (hunk 1 above)
Comment (davidplumb) [reply example_runthroughs.md]: fixed

File: example_runthroughs.md (hunk 5 above)
Comment (davidplumb) [reply example_runthroughs.md]: done

File: example_runthroughs.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+|2003|6|Scotland|S92000003|false|
+|2003|7|Wales|W92000004|false|
+
+The user now selects the entirety of Wales from the above selection of top level geographies. We then query the `geography_areas` table for the corresponding area data.
+
+```sql
+SELECT * 
+FROM c2011_meta.geography_areas
+WHERE c2011_meta.geography_areas.geography_grouping_id = 2003
+AND c2011_meta.geography_areas.top_level_geography_id = 7;
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: SQL style guide spacing needs adding in this and other queries.

File: example_runthroughs.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+|-|-|-|-|-|
+|2000|1|United Kingdom|K02000001|false|
+|2001|2|Great Britain|K03000001|true|
+|2002|3|England and Wales|K04000001|true|
+|2003|4|England|E92000001|false|
+|2003|5|Northern Ireland|N92000002|false|
+|2003|6|Scotland|S92000003|false|
+|2003|7|Wales|W92000004|false|
+
+The user now selects the entirety of Wales from the above selection of top level geographies. We then query the `geography_areas` table for the corresponding area data.
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: top level > top-level

File: example_runthroughs.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+|QS601SC0010|{AGE:46,ECOACT:570,UNIT:1962}|Age 16 to 74 // Economically active\ Full-time students // Persons|QS601_0_SC_MRG_RCD_AGG|
+|QS601SC0013|{AGE:46,ECOACT:3429,UNIT:1962}|Age 16 to 74 // Persons // Economically inactive\ Student|QS601_0_SC_MRG_RCD_AGG|
+|QS601UK0003|{AGE:46,ECOACT:3823,UNIT:1962}|Age 16 to 74 // Persons // Economically active\ Employee\ Part-time (excluding full-time students)|QS601_0_UK_MRG_RCD_AGG|
+|QS601UK0004|{AGE:46,ECOACT:3795,UNIT:1962}|Age 16 to 74 // Persons // Economically active\ Employee\ Full-time (excluding full-time students)|QS601_0_UK_MRG_RCD_AGG|
+
+
+To get the number of citizens each `name` and `table_name` this corresponds to we can do a query like so:
+
+```sql
+SELECT geocodeid, QS601SC0010 from c2011.QS601_0_SC_MRG_RCD_AGG;
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: SQL style guide as mentioned for other queries, and capitalise `FROM`

File: example_runthroughs.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+|geocodeid|QS601SC0010|
+|-|-|
+|354549|78|
+|354550|90|
+|354551|81|
+|354552|63|
+|354553|120|
+
+The `geocodeid` referes back to the `geography_area_relations` `child_id` column.
```
Comment (tomgiddingsjisc) [inline example_runthroughs.md]: referes > refers

**Review by tomgiddingsjisc — COMMENTED** [review]
A couple of things - also, please can you add a link from [README.md](https://github.com/JiscSD/ukcas-documentation/blob/issue-15-Usage_examples/README.md) in the root directory under a header of 'Usage examples` or similar?

**Review by tomgiddingsjisc — APPROVED** [review]

**Pull request merged**